src/types.ts

```diff
@@ -72,3 +72,8 @@ export type AdsConfigEnabled =
 export type AdsConfig = AdsConfigEnabled | AdsConfigDisabled;
 
 export type AdTargetingBuilder = () => Promise<AdsConfig>;
+
+type True = 't';
+type False = 'f';
+type NotApplicable = 'na';
```

Comment:
Where is this used? I think it might Not be Applicable to this PR 😉

Reply:
Never mind! I see it's used in #422

Reply:
Yeah, I’ve just cherry-picked the same commit in all of them.